client/src/pages/ChangePasswordPage.jsx

```diff
@@ -0,0 +1,166 @@
+      <section>
+        <p>
+          Success. Your password has been changed.
+          <Link to={'/login'}>Log in</Link>
+        </p>
+      </section>
+    );
+  }
+
+  return (
+    <>
+      <Formik
+        initialValues={{
+          password: '',
+          confirmation: '',
+        }}
+        validateOnMount={true}
+        onSubmit={({ password, confirmation }, formikHelpers) => {
+          formikHelpers.setSubmitting(true);
+
+          authService
+            .changePassword({ password, confirmation, resetToken })
+            .then(() => {
+              setChanged(true);
+            })
+            .catch((error) => {
+              if (error.message) {
+                setError(error.message);
+              }
+
+              if (!error.response?.data) {
+                return;
+              }
+
+              const { errors, message } = error.response.data;
+
+              formikHelpers.setFieldError('password', errors?.password);
+              formikHelpers.setFieldError('confirmation', errors?.confirmation);
+
+              if (message) {
+                setError(message);
+              }
+            })
+            .finally(() => {
+              formikHelpers.setSubmitting(false);
+            });
+        }}
+      >
+        {({ touched, errors, isSubmitting }) => (
+          <Form className="box">
+            <h1 className="title">Change password</h1>
+            <div className="field">
+              <label htmlFor="new-password" className="label">
+                New password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
+                  name="password"
+                  type="password"
+                  id="new-password"
+                  placeholder="*******"
+                  className={cn('input', {
+                    'is-danger': touched.password && errors.password,
+                  })}
+                />
+
+                <span className="icon is-small is-left">
+                  <i className="fa fa-lock"></i>
+                </span>
+
+                {touched.password && errors.password && (
+                  <span className="icon is-small is-right has-text-danger">
+                    <i className="fas fa-exclamation-triangle"></i>
+                  </span>
+                )}
+              </div>
+
+              {touched.password && errors.password ? (
+                <p className="help is-danger">{errors.password}</p>
+              ) : (
+                <p className="help">At least 6 characters</p>
+              )}
+            </div>
+
+            <div className="field">
+              <label htmlFor="confirm-password" className="label">
+                Confirm password
+              </label>
+
+              <div className="control has-icons-left has-icons-right">
+                <Field
+                  validate={validatePassword}
```

Comment:
The `validatePassword` function is used for both the `password` and `confirmation` fields. However, it only checks for the length of the password. You might want to add a check to ensure that the `confirmation` field matches the `password` field.